docs/registering-samples.md

```diff
@@ -22,18 +22,37 @@ The basic set of data that must be registered for each sample consists of:
 
 During sample registration, **ARGO IDs** will be assigned to your program data. Any attempts to submit data that does not refer to a registered sample will result in an error. You will be prompted to complete sample registration before any clinical or molecular data is submitted to your program.
 
-## Registering Samples
+## Downloading and Formatting the Sample Registration File
+
 1. Navigate to the **Submission** area in the top menu.
 1. Click on the **Register Samples** section in the left menu for your program.
-1. Download a **TSV Template** of the sample registration file and format the file according to the current [Data Dictionary](/dictionary) specifications.
+1. Download the **TSV Template** for the sample registration file and format it according to the current [Data Dictionary](/dictionary) specifications.
+
+For help with formatting this file, please refer to [Tips for Formatting your TSV files]( submitting-clinical-data#tips-for-formatting-your-tsv-files)
+
+![Download and Format File](/assets/submission/register-1-download.png)
+
+## Uploading Registration TSV File
+![Upload Files](/assets/submission/register-2-upload.png)
+
 1. Once your file is formatted, click the **Upload File** button and select your file from the browser. Only TSV file types are supported, and the file name must begin with the *sample-registration* and end with _.tsv_.
 1. Upon uploading, if there are any errors in your file they will be displayed within the Sample Registration workspace. The error report will also be available for download. You must fix all of the errors that are listed within your sample registration file and then reupload it.
 1. Valid files will be available for preview in the Sample Registration workspace.  You can review the new samples (purple star) versus previously registered samples (grey star) by filtering the table on the star column.
+
+## Registering Samples
+![Register Samples](/assets/submission/register-3-register.png)
+
 1. Once you have reviewed the file preview, click on the **Register Samples** button to submit your samples.
 
-Once registered, donors, along with specimen and sample data, will be visible on your [Program Dashboard](/docs/submitted-data).
+Once registered, donors, along with specimen and sample counts, will be visible on your [Program Dashboard](/docs/submitted-data).
 
 ## Correcting Already Registered Data
 Once samples are registered and data processing and analysis proceeds, it can be difficult to correct the data manually.
 
-If you have made an error to registered sample data, please [contact the DCC](https://platform.icgc-argo.org/contact) and they will assist in correcting your registered data.  
+If you have made an error with registered sample data, please [contact the DCC](https://platform.icgc-argo.org/contact) and they will assist in correcting your registered data. 
+
+## Multiple Data Submitters 
```

Comment:
Do you think this section should go at the top instead?

Reply:
I can make this change in a new branch after this PR is approved

Reply:
approved!